datarepo-clienttests/src/main/resources/configs/basicexamples/FailurePatternExample.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "FailurePatternExample",
+  "description": "Example use of failure scripts during bulk load",
```

Comment:
Nit: Could you elaborate on what the test is doing? I *think it's running 3 bulk file loads each on 2 different datasets, while kiilling a random pod after ~30 seconds. Is that right?

Reply:
Yes, you're right! I adjusted the test slightly and updated the description.